docs/configuration.md

```diff
@@ -47,51 +47,51 @@ Iceberg tables support table properties to configure table behavior, like the de
 
 ### Write properties
 
-| Property                                             | Default                     | Description                                                                                                                                                                                       |
```

Comment:
@tomtongue thanks for updating this. Can you please update the PR so that there's a diff only for that one single line that needs the change?

Reply:
@nastra thanks for quickly reviewing this. And sorry for adding unnecessary spaces (I missed they were automatically added by the editor). Remove the spaces.